jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanel.kt

```diff
@@ -54,7 +64,13 @@ class SchemaSelectionPanel : WizardFragment {
         return null
     }
 
-    override fun isApplicable(template: SamProjectTemplate?): Boolean = template?.supportsDynamicSchemas() == true
+    override fun isApplicable(template: SamProjectTemplate?): Boolean {
+        val supported = template?.supportsDynamicSchemas() == true
+        if (supported && !selectorInitialized.getAndSet(true)) {
+            schemaSelector.reload()
+        }
```

Comment:
hmmm...do we need to reload it? The schemas aren't going to change since we aren't changing the creds.